Code/Tools/TestImpactFramework/Runtime/Code/Include/TestImpactFramework/TestImpactClientSequenceReport.h

```diff
@@ -122,12 +122,14 @@ namespace TestImpact
         };
 
         //! Base class for all sequence report types.
-        template<typename PolicyStateType>
+        template<SequenceReportType Type, typename PolicyStateType>
         class SequenceReportBase
         {
         public:
+            static constexpr SequenceReportType m_type = Type;
```

Comment:
nit: As this is static we should avoid using `m_`, this could just by CamelCase I think these days

Reply:
fixed 👍